src/components/Hamburger/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from "react";
+import HamburgerMenu, { HamburgerMenuProps } from "react-hamburger-menu";
+import styles from "./Hamburger.module.css";
+
+const Hamburger: FC<HamburgerMenuProps> = ({
+  isOpen,
+  height,
+  width,
+  strokeWidth,
+  color,
```

Comment:
skal color ha default verdi så slipper man å måtte sende med med mindre man skulle hatt bruk for å ha krysset i en annen farge?